I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/card-match-game.rpy", line 165, in script call
    call memoria_game
  File "game/card-match-game.rpy", line 120, in script
    $ result = ui.interact()
  File "game/card-match-game.rpy", line 120, in <module>
    $ result = ui.interact()
OSError: Couldn't find file 'audio/cafe.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/card-match-game.rpy", line 165, in script call
    call memoria_game
  File "game/card-match-game.rpy", line 120, in script
    $ result = ui.interact()
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/card-match-game.rpy", line 120, in <module>
    $ result = ui.interact()
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\display\core.py", line 2166, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\display\core.py", line 3026, in interact_core
    renpy.audio.audio.periodic()
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\audio\audio.py", line 1199, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\audio\audio.py", line 1177, in periodic_thread_main
    periodic_pass()
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\audio\audio.py", line 1119, in periodic_pass
    c.periodic()
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\audio\audio.py", line 528, in periodic
    topf = load(filename)
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\audio\audio.py", line 72, in load
    rv = renpy.loader.load(fn, directory="audio")
  File "C:\Users\user\Downloads\renpy-8.2.3-sdk\renpy\loader.py", line 672, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'audio/cafe.mp3'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.2.3.24061702
secret recipe 1.0
Sat Jul 13 17:13:53 2024
